src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json

```diff
@@ -39,7 +39,7 @@
     },
     {
         "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask",
-        "version": "2.4.2",

```

Comment:
Can you also make the change in v3.x branch

Reply:
Yes, here's the PR: #77.